src/newcode/newCodeDefinitionService.ts

```diff
@@ -48,18 +48,18 @@ export class NewCodeDefinitionService {
 
   createNewCodeDefinitionStatusBarItem(context: VSCode.ExtensionContext) {
     context.subscriptions.push(VSCode.commands.registerCommand(Commands.NEW_CODE_DEFINITION, () => {
-      VSCode.window.showQuickPick([{ label: `Toggle 'Clean as You Code'` }, { label: `Learn more about 'Clean as You Code'` }])
+      VSCode.window.showQuickPick([{ label: `Toggle 'SonarLint focus'` }, { label: `Learn more about 'Clean as You Code'` }])
```

Comment:
According to [this doc](https://docs.google.com/document/d/18-Nk3iG0W0lvDr6D904evJJM0LzJYBSJUn96TBC4YXE/edit), this label should say '[Learn how](https://docs.sonarsource.com/sonarlint/vs-code/using-sonarlint/investigating-issues/#focusing-on-new-code) to deliver clean code with Clean as You Code.'